src/index.ts

```diff
@@ -19,35 +19,38 @@ export default {
 		patches.push(unsubscribeFromUserDecorationsStore);
 		patches.push(
 			after('getUser', UserStore, (_, ret) => {
-				if (ret && !ret.avatarDecoration?.startsWith('decor_') && users?.has(ret.id)) ret.avatarDecoration = `decor_${users?.get(ret.id)}`;
+				if (ret && !ret.avatarDecoration?.asset?.startsWith('decor_') && users?.has(ret.id)) ret.avatarDecoration = {
+					asset: `decor_${users?.get(ret.id)}`,
+					skuId: "0"
+				};
 			})
 		);
 
 		patches.push(
 			after('getAvatarDecorationURL', ImageResolver, ([{ avatarDecoration, canAnimate }], _) => {
-				if (avatarDecoration?.startsWith('decor')) {
-					const parts = avatarDecoration.split('_').slice(1);
+				if (avatarDecoration?.asset?.startsWith('decor')) {
+					const parts = avatarDecoration.asset.split('_').slice(1);
 					if (!canAnimate && parts[0] === 'a') parts.shift();
 					return CDN_URL + `/${parts.join('_')}.png`;
-				} else if (avatarDecoration?.startsWith('file://')) {
-					return avatarDecoration;
+				} else if (avatarDecoration?.asset?.startsWith('file://')) {
+					return avatarDecoration.asset;
 				}
 			})
 		);
 
 		patches.push(
 			after('isAnimatedAvatarDecoration', AvatarDecorationUtils, ([avatarDecoration], _) => {
-				if (ReactNative.Platform.OS === 'ios' && avatarDecoration?.startsWith('file://')) return true;
+				if (ReactNative.Platform.OS === 'ios' && avatarDecoration?.asset?.startsWith('file://')) return true;
 			})
 		);
 
-		patches.push(
-			(() => {
-				const oldVal = CollectiblesExperiment.getCurrentConfig().canUseAvatarDecorations;
-				CollectiblesExperiment.getCurrentConfig().canUseAvatarDecorations = true;
-				return () => (CollectiblesExperiment.getCurrentConfig().canUseAvatarDecorations = oldVal);
-			})()
-		);
+		// patches.push(
+		// 	(() => {
+		// 		const oldVal = CollectiblesExperiment.getCurrentConfig().canUseAvatarDecorations;
+		// 		CollectiblesExperiment.getCurrentConfig().canUseAvatarDecorations = true;
+		// 		return () => (CollectiblesExperiment.getCurrentConfig().canUseAvatarDecorations = oldVal);
+		// 	})()
+		// );
```

Comment:
genuienly have no clue how to replace this

Reply:
however even with no experiments enabled avatar decorations appear, so i think i can delete this